src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -255,8 +255,6 @@ def __get_patch_id(self, package_name, package_version):
 
     def get_os_name_and_version(self):
         try:
-            if self.env_layer.platform.system() != "Linux":
-                raise Exception("Unsupported OS type: {0}.".format(self.env_layer.platform.system()))
```

Comment:
Why have we removed this?

Reply:
This will hit when it's run on Windows. When on Windows, regardless of whether these two lines exist or not, this function will always behave the same (an exception will be raised and caught saying 'Unable to determine platform information', and the return will have the same value. The only difference between old and new code is the additional text 'Unsupported OS type:'.

Generally, this kind of code is good to retain but having this come up is never a mainline customer-facing scenario. It may be a dev scenario (bad test code), but devs should be able to understand that simply from line 261. Given these circumstances and that we have to go out of our way to get test coverage for almost zero utility lines, stripped them out.